src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,19 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private int dippingDepth;
```

Comment:
The field `dippingDepth` should be private to prevent direct access to it. Use getters and setters instead.